src/main/resources/META-INF/resources/seasons.js

```diff
@@ -42,6 +44,19 @@ const select = id => {
     }
 }
 
+const summaries = document.getElementsByClassName("summary")
+Array.from(summaries).forEach(summary => {
+    summary.style.height = 100 - summary.parentElement.children[0].scrollHeight - summary.parentElement.children[1].scrollHeight - summary.parentElement.children[2].scrollHeight - 14 + "px"
```

Comment:
We should see if we can introduce variable for the 100 so that we understand later that it is the initial height as the basis of the calculation.